### PR TITLE
Deprecate '.feature' extension

### PR DESCRIPTION
**Pull request by wirednkod**
Replace `.feature` extension with `.zndsl` as discussed and agreed [here](https://github.com/paritytech/zombienet-vscode-extension/issues/7#issuecomment-1257669012)

### CODE REVIEW

File: docs/src/cli/test-dsl-definition-spec.md [hunk 1]
```diff
@@ -12,18 +13,17 @@ One of the goals of Zombienet it to provide a simple way to create tests, for th
 - Custom api calls (through polkadot.js)
 - Commands
 
-Each of this abstractions are expresed by sentences defined in a *natural languaje style*, so each test line will be mapped to a test tu run.
+Each of this abstractions are expresed by sentences defined in a **natural languaje style**, so each test line will be mapped to a test tu run.
```
Comment (chevdor) [inline docs/src/cli/test-dsl-definition-spec.md:16]:
```suggestion
Each of this abstractions are expresed by sentences defined in a **natural language style**, so each test line will be mapped to a test tu run.
```

File: README.md [hunk 2]
```diff
@@ -261,7 +261,7 @@ You can teardown the network (and cleanup the used resources) by terminating the
 The other goal of `zombienet` is to provide a way to perform test/assertions against the spawned
 network, using a set of `natural language expressions` that allow you to make assertions based on
 metrics, logs and some `built-in` function that query the network using `polkadot.js`. Those
-assertions should be defined in a *feature test*, and the `dsl` and format is documented in
+assertions should be defined in a *.zndsl test*, and the `dsl` and format is documented in
```
Comment (chevdor) [inline README.md:264]:
```suggestion
assertions should be defined in a *.zndsl test*, and the `dsl` (**D**omain **S**pecific **L**anguage) and format is documented in
```

**Review by chevdor — APPROVED** [review]

File: docs/src/cli/test-dsl-definition-spec.md (hunk 1 above)
Comment (wirednkod) [reply docs/src/cli/test-dsl-definition-spec.md:16]: Good catch. Thank you

File: README.md (hunk 2 above)
Comment (wirednkod) [reply README.md]: Nice one! Thank you

**Review by pepoviola — APPROVED** [review]
Looks good, thanks!!

**Pull request merged**